### ISSUE TITLE
install the man page

### ISSUE THREAD

**Issue by andychu** [oilshell/oil#418]
Use `$prefix/share/man/man1`

Do we need to gzip it?

**Comment by The-King-of-Toasters** [comment]
> Use `$prefix/share/man/man1`

I recommend doing something like `$DESTDIR$PREFIX$MANPREFIX/man1`,
where `DESTDIR=""`, `$PREFIX="/usr/local"` and `$MANPREFIX="/man"` by default.
These can be set differently according to a configure script/user input.
An example install script could look like:

```sh
#!/bin/sh
DESTDIR="${DESTDIR:-}"
PREFIX="${PREFIX:-/usr/local}"
MANPREFIX="${MANPREFIX:-/man}"

mkdir -p "$DESTDIR$PREFIX$MANPREFIX/man1/"
cp "doc/osh.1" "$DESTDIR$PREFIX$MANPREFIX/man1/"
```
> Do we need to gzip it.

No, this is handled by the packager/packaging system. I know that Arch automatically gzips any files that are installed to `$pkgdir/usr/share/man`, whilst OpenBSD does not.

**Comment by The-King-of-Toasters** [comment]
I'd like to implement this, but I'm having a bit of trouble compiling from git to test it out. Are there any docs that document how this is done?

**Comment by andychu** [comment]
The tarball process is documented under "three kinds of builds" here:

https://github.com/oilshell/oil/wiki/Contributing

Although I didn't see the justification/logic behind your suggestion.  I'd rather implement something considerably simpler and then expand it if anyone complains.  It's not clear who uses those variables.

I planned to take a stab at it myself.  I guess I'm more interested in the test results than the patch... i.e. how many systems would we be able to get `man osh` working on?

**Comment by andychu** [comment]
Released with 0.7.pre2: https://www.oilshell.org/release/0.7.pre2/

Documented it a bit here: https://www.oilshell.org/release/0.7.pre2/doc/INSTALL.html

If there are problems installing the man page on a particular platform, please file a bug with repro steps.  Thanks!
